### ISSUE TITLE
Accessibility: Limited navigation and missing focus highlighting with screen readers

### ISSUE THREAD

**Issue by georgwittberger-t-systems-mms** [nathanreyes/v-calendar#1181]
Current behavior:

When using a screen reader in a date picker popover it is possible to step through days but calendar does not switch to previous or next month when trying to navigate past the boundaries of current month (e.g. pressing arrow up in first week, which should switch to previous month).

Also, focus styling is not applied anymore when navigating popover calendar with a screen reader.

Expected behavior:

Date picker popover can be navigated with keyboard as usual when using a screen reader.

Further notes:

One solution we found for this is setting attribute `role="application"` on popover content element `.vc-popover-content`. This tells screen readers to step out of browsing mode and treat this area like a pure desktop application. Since VCalendar manages focus properly when using arrow keys this should be fine.

In addition it would be desirable to set ARIA attribute `aria-label` on popover content element (which has application role) to announce the purpose of the application. The value could be passed using a new prop on `v-date-picker`.
